index.js

```diff
@@ -3,6 +3,11 @@ import postcss from 'postcss';
 import perfectionist from 'perfectionist';
 import postcssSafeParser from 'postcss-safe-parser';
 
+const SUPPORTED_SCOPES = [
```

Comment:
I copied this from your code https://github.com/sindresorhus/atom-esformatter/blob/b7eee38f1c51a0a4cbb96c0cb79862a72746def9/index.js#L5

It's a `const` there.